assets/sass/02-tools/extends.scss

```
@@ -1,4 +1,5 @@
-// Button Placeholder style
+/* Button extends */
```

Do these comments need to make it into the compiled CSS? 
```suggestion
// Button extends
```

---

Without them in the css file the code felt out of place because there is no heading or explaination to why the button styles are loaded this early.